lib/galaxy/tool_util/deps/mulled/mulled_build.py

```diff
@@ -341,7 +339,12 @@ def __init__(
     ):
         if not conda_exec:
             conda_image = CONDA_IMAGE or "continuumio/miniconda3:latest"
-            conda_exec = docker_command_list("run", [conda_image, "conda"])
+            binds = []
+            for channel in ensure_channels:
+                if channel.startswith("file://"):
+                    bind_path = channel[7:]
+                    binds.extend(["-v", f"/{bind_path}:/{bind_path}"])
```

Comment:
How come you're prefixing the bind paths with `/`, and wouldn't you want to to hardcode the bind destination to what the container expects ?

Reply:
> How come you're prefixing the bind paths with /, 

I took inspiration from here: https://github.com/galaxyproject/galaxy/blob/682b67e97710dbb7a6ec54fa6df08d2c95d7bac3/lib/galaxy/tool_util/deps/mulled/mulled_build.py#L247

But you are right the `/` is redundant. I will remove it at both places.

> and wouldn't you want to to hardcode the bind destination to what the container expects ?

This should be what the container expects, i.e. the same value is given to the `-c / --channel` argument of `conda search`.

Reply:
I see, that makes sense.